pages/press/index.js

```diff
@@ -0,0 +1,365 @@
+          This programming system covers a hitherto unexplored field. 
+          There is no direct competition as it is inserted in a new and emerging market.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          The project is based on transparency, teamwork, and passion, cooperation with other free software initiatives. 
+          Additionally, AsyncAPI can't be understood without the talent and the contributions of its community. 
+          It has been crucial for getting to this point.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          Inspired by the logic of OpenAPI, in less than five years it has managed to carve out a recognized position for itself in the world of APIs. 
+          This project is seeking to improve the current state of event-driven architectures (EDA).
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          And, although it was born out of the need to fill a gap in the standardization of asynchronous APIs, the differential value of the initiative is that it has created its sense of purpose and has grown without a profit motive.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          One of the main challenges is to make the development experience super seamless, from idea to production. 
+          Users should be able to create their first API within minutes without prior knowledge of AsyncAPI, the communication protocol, or anything else that's not business logic.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          Right now, the strategy of the project is to continue improving the specification, and implementing resources that make it more complete and solid. 
+          This has led to the partnership with Postman (December 2020), a leading collaborative platform for API development and a reference within this sector all around the world. 
+          This partnering allows it to boost the development of more and better tools to help engineers create and maintain Asynchronous APIs while using their favorite programming languages and frameworks.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          The next step has been to host the project on a neutral foundation to ensure the long-term sustainability and success of the initiative. AsyncAPI has been hosted by the Linux Foundation since March of 2021.
+        </Paragraph>
+        <Paragraph typeStyle="body-md" className="my-4">
+          However, none of the other goals are possible without the support of a large community. 
+          AsyncAPI is and must remain a community-driven initiative.
+        </Paragraph>
+        
+        <Heading level="h3" typeStyle="heading-md" className="my-5">
+          <span id="asyncapiInNumbers">AsyncAPI, in numbers</span>
```

Comment:
do we really need this section? it has old numbers really. We also talk about 400% increase but do not really say what is the base number 😄 

if you really believe it is needed then link to 2 articles I wrote that summarize numbers for 2020 and 2021